docs/source/reference/kubernetes/index.rst

```diff
@@ -0,0 +1,152 @@
+.. _kubernetes-overview:
+
+Deploying on Kubernetes (Alpha)
```

Comment:
Deploying sounds like "deploying skypilot (as a long-running service)". Wdyt about "Running on Kubernetes"?

Reply:
I'm ambivalent between "Deploying on Kubernetes" and "Running on Kubernetes". I've changed to "Running on Kubernetes".

Personally, I prefer "SkyPilot on Kubernetes", since it captures both - deployment (initial setup and considerations involved in SkyPilot on Kubernetes) and running (ongoing tasks, benefits, and features of using SkyPilot with Kubernetes) tasks. Ray also adopts a similar title ["Ray on Kubernetes"](https://docs.ray.io/en/latest/cluster/kubernetes/index.html).